Nodejs/Product/Nodejs/Project/MigrateToJspsInfoBar.cs

```diff
@@ -33,6 +44,11 @@ void OnClose()
             {
                 CurrentInfoBarElement.Unadvise(eventCookie); 
             }
+
+            void MigrateProjectCommand()
+            {
+                NodejsPackage.Instance.DTE.Commands.Raise(Guids.MigrateToJspsCmdSet.ToString(), PkgCmdId.cmdidJspsProjectMigrate, null, null);
```

Comment:
I would have expected there to be a project id passed in somewhere, but since there isn't, I'm wondering if this will work with multiple projects? Would multiple goldbars show up (one for each project)?

Reply:
Nope, just 1 goldbar